app/brave_settings_strings.grdp

```diff
@@ -1439,6 +1439,18 @@ Leo. This action cannot be undone.
   <message name="IDS_SETTINGS_LEO_ASSISTANT_CLEAR_HISTORY_DATA_SUBLABEL" desc="The text for settings option">
     Data and chat history
   </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_MODEL_SELECTION_LABEL" desc="Label for selecting default model">
+    Default model for new conversations
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_CHAT_DEFAULT_SUBTITLE" desc="A description for default chat model">
+    General purpose chat
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_CHAT_LEO_EXPANDED_SUBTITLE" desc="A description for llama-2-70b model">
+    Advanced and accurate chat
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_CHAT_CLAUDE_INSTANT_SUBTITLE" desc="A description for claude-instant-v1 model">
+    Strength in creative tasks
+  </message>
```

Comment:
Instead of duplicating these strings her and in ai_chat_ui_strings.grdp, can we reference the IDS_CHAT_UI_XYZ ids in brave_settings_localized_strings_provider.cc

Reply:
done